pkgs/development/tools/yarn2nix-moretea/yarn2nix/bin/yarn2nix.js

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env node
```

Comment:
Would be nice to mention how it was formatted in the commit message.

I believe the choice of single quotes and omission of return values was intentional but if we can rely on auto-formatter, that will allow us to write code without having to think about low level things like syntax so :+1: from me.

Reply:
I just used the `yarn format` script in the package.json. I completely agree, I don't really care what the code style is, as long as it's encoded into a format command and enforced so the code looks good and you don't have to think about it.